examples/runner-default/main.tf

```diff
@@ -40,6 +40,10 @@ module "runner" {
   runners_name             = var.runner_name
   runners_gitlab_url       = var.gitlab_url
   enable_runner_ssm_access = true
+  cache_bucket = {
```

Comment:
Maybe I am missing something, but why adding this object to the example. What is causing this need?

Reply:
Seems that this is from one of my other branches. I will remove it here.